tools/apiview/parsers/js-api-parser/.npmrc

```diff
@@ -1,3 +1,3 @@
-registry=https://registry.npmjs.org/
-@azure-tools:registry=https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-js/npm/registry/
-@azure-tools:always-auth=true
\ No newline at end of file
+registry=https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-js/npm/registry/
```

Comment:
Does this break local auth for installs?

Reply:
I have updated to make it scope based so it uses feed only for @azure-tools package.  Local installation work fine.
Permission issue happens when we attempt to publish from pipeline. We can publish the artifact from local machine.